### PR TITLE
Pin vs2019

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by h-vetinari** [comment]
I think @hmaarrfk mentioned that this was running into: https://github.com/conda-forge/clangdev-feedstock/issues/202

Will hopefully be fixed soon.

**Comment by jschueller** [comment]
good to know, thanks

File: conda-forge.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ azure:
   settings_win:
     variables:
       CONDA_BLD_PATH: C:\bld\
+    pool:
```
Comment (hmaarrfk) [inline conda-forge.yml:5]: I did this manually. U didn't know about this at the time

**Pull request merged**